test/autotest.py

```diff
@@ -860,7 +846,7 @@ def saveResultsNMI():
 # program will try to unlink the file once again, but the unlink operation will
 # fail, causing the test to fail.
 old_ckpt_cmd = CKPT_CMD
-CKPT_CMD = b'Kc' # Equivalent to 'dmtcp_command -kc'
+CKPT_CMD = "kc" # Equivalent to 'dmtcp_command -kc'
```

Comment:
Same here. We don't need to break Python 2.7 for no special reason.
'Kc' and "Kc" are equivalent. But removing b is not equivalent.
Please do: b'Kc' -> b"Kc".

Also, why change `Kc` to `kc` (lower-case)?